src/web/app/pages-session/session-submission-page/session-submission-page.component.ts

```diff
@@ -311,6 +311,15 @@ export class SessionSubmissionPageComponent implements OnInit, AfterViewInit {
         }
 
         this.loadFeedbackQuestions();
+
+        // Display note on submission on mobile device
+        if (this.feedbackSessionSubmissionStatus === FeedbackSessionSubmissionStatus.OPEN && window.innerWidth < 768) {
```

Comment:
Could you refactor `768` into a descriptive variable? Future developers might not understand where the value is coming from. You could even add a short comment that the threshold comes from the corresponding `scss` file.